vim/vscode-code-actions.vim

```diff
@@ -93,5 +93,9 @@ xnoremap <C-w>gd <Cmd>call VSCodeNotify('editor.action.revealDefinitionAside')<C
 xnoremap <expr> <C-/> <SID>vscodeCommentary()
 nnoremap <expr> <C-/> <SID>vscodeCommentary() . '_'
 
+" open quickfix menu for spelling corrections and refactoring
+" only keyboard arrows can be used to navigate, for a solution, see https://github.com/asvetliakov/vscode-neovim#keyboard-quickfix
+nnoremap z= <Cmd>call VSCodeNotify('editor.action.quickFix')<CR>
```

Comment:
Not sure this is something of universal use. Users can remap this easily in their local installation.

Reply:
While I do agree, I think that it does not hurt to have some default functionality for a common keybinding, rather than it not working at all.